man/systemctl.xml

```diff
@@ -882,17 +882,16 @@ kobject-uevent 1 systemd-udevd-kernel.socket systemd-udevd.service
           <term><command>show</command> <optional><replaceable>PATTERN</replaceable>...|<replaceable>JOB</replaceable>...</optional></term>
 
           <listitem>
-            <para>Show properties of one or more units, jobs, or the
-            manager itself. If no argument is specified, properties of
-            the manager will be shown. If a unit name is specified,
-            properties of the unit are shown, and if a job ID is
-            specified, properties of the job are shown. By default, empty
-            properties are suppressed. Use <option>--all</option> to
-            show those too. To select specific properties to show, use
-            <option>--property=</option>. This command is intended to be
-            used whenever computer-parsable output is required. Use
-            <command>status</command> if you are looking for formatted
-            human-readable output.</para>
+            <para>Show properties of one or more units, jobs, or the manager itself. If no argument is specified,
+            properties of the manager will be shown. If a unit name is specified, properties of the unit are shown, and
+            if a job ID is specified, properties of the job are shown. By default, empty properties are suppressed. Use
+            <option>--all</option> to show those too. To select specific properties to show, use
+            <option>--property=</option>. This command is intended to be used whenever computer-parsable output is
+            required. Use <command>status</command> if you are looking for formatted human-readable output.</para>
+
+            <para>Many properties shown with this command are closely related to the settings of the system and service
+            manager and its unit files. However, the properties shown by <command>systemctl show</command> are
+            generally more low-level, normalized, and cover both configuration and runtime state.</para>
```

Comment:
To be honest, I don't find this very legible. "this command" *is* `show`, right? The "however" sounds like describing two opposing sides. What are "settings" of the system manager/unit files, configuration or state?

Just having the second sentence (without "However, " and "more") would be much clearer IMHO.